PollBuddy-Server/backend/routes/users.js

```diff
@@ -661,6 +663,226 @@ router.post("/me/groups", function (req, res) {
   return sendResponse(res, httpCodes.MethodNotAllowed("POST is not available for this route. Use GET."));
 });
 
+/**
+ * This route is not used. It is simply there to have some response to /api/users/forgotpassword/ when using GET
+ * @getdata {void} None
+ * @postdata {void} None
+ * @returns {void} Status 405 { "result": "failure", "error": "Route not availible."}
+ * @name backend/users/forgotpassword_GET
+ * @param {string} path - Express path
+ * @param {callback} callback - function handler for route
+ */
+router.get("/forgotpassword/",function (req,res) {
+  return res.status(405).send(createResponse(null,"Route not availible."));
```

Comment:
I think we have a new thing for this, httpcodes.unavailable or something like that

Reply:
There's a couple like that